.github/workflows/frontend-check.yaml

```diff
@@ -0,0 +1,55 @@
+name: Cypress tests
+on: 
+  push:
+    # branches:
+    #   - main
+    # tags:
+    #   - v*
+  pull_request:
+jobs:
+  install:
+    runs-on: ubuntu-24.04
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Cypress install
+        uses: cypress-io/github-action@v6
+        with:
+          # Disable running of tests within install job
+          runTests: false
+          build: npm run build
```

Comment:
Interesting ... it's complaining about not finding a `.lock` file. Does the `project` tag below not acting as a working directory??